server/views/components/sessionTemplateVisitStats.test.ts

```diff
@@ -0,0 +1,46 @@
+import * as cheerio from 'cheerio'
+import nunjucks from 'nunjucks'
+import { registerNunjucks } from '../../utils/nunjucksSetup'
+import { SessionTemplateVisitStatsDto } from '../../data/visitSchedulerApiTypes'
+import TestData from '../../routes/testutils/testData'
+
+const njkEnv = registerNunjucks(null)
+const nunjucksString = `
+  {%- from "components/sessionTemplateVisitStats.njk" import sessionTemplateVisitStats -%}
+  {{ sessionTemplateVisitStats(visitStats) }}
+`
+const compiledTemplate = nunjucks.compile(nunjucksString, njkEnv)
+
+let visitStats: SessionTemplateVisitStatsDto
+const viewContext = { visitStats }
+
+describe('sessionTemplateVisitStats(visitStats) macro', () => {
```

Comment:
Testing the macro directly to avoid to simplify and avoid duplication in the routes tests for view / edit a template.